content/en/blog/_posts/2021-04-29-SIG-Usability-Spotlight.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+title: "SIG-Usability Spotlight"
+date: 2021-04-29
+slug: sig-usability-spotlight-2021
+---
+
+**Author:** [Kunal Kushwaha](https://twitter.com/kunalstwt)
+
+## Introduction
+
+Are you interested in learning about what [SIG-Usability](https://github.com/kubernetes/community/tree/master/sig-usability) does and how you can get involved? Well, you're at the right place. SIG-Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community. In this blog, we have summarized our conversation with [Gaby Moreno](https://twitter.com/morengab) who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.
+
+Gaby is a co-lead for SIG-Usability. She works as a Product Designer at IBM and enjoys working on the user experience of open, hybrid cloud technologies like Kubernetes, OpenShift, Terraform, and Cloud Foundry.
+
+## A brief summary of our converstation is captured below
+
+### Q. Could you tell us a little about what SIG-Usability does?
+
+A. SIG-Usability at a high level started because there was no dedicated user experience for Kubernetes. The extent of SIG convenience is the centre of end-client ease of use of the Kubernetes project. The main activity is user research for the community. Which includes speaking to people that use Kubernetes. 
```

Comment:
```suggestion
A. SIG-Usability at a high level started because there was no dedicated user experience for Kubernetes. The extent of SIG convenience is the centre of end-client ease of use of the Kubernetes project. The main activity is user research for the community, which includes speaking to Kubernetes users. 
```